velox/type/Timestamp.h

```diff
@@ -76,11 +76,9 @@ struct Timestamp {
 
   Timestamp(int64_t seconds, uint64_t nanos)
       : seconds_(seconds), nanos_(nanos) {
-    VELOX_USER_DCHECK_GE(
-        seconds, kMinSeconds, "Timestamp seconds out of range");
-    VELOX_USER_DCHECK_LE(
-        seconds, kMaxSeconds, "Timestamp seconds out of range");
-    VELOX_USER_DCHECK_LE(nanos, kMaxNanos, "Timestamp nanos out of range");
+    VELOX_USER_CHECK_GE(seconds, kMinSeconds, "Timestamp seconds out of range");
```

Comment:
Don't we have this code running in hot loops? Will this affect performance?

CC: @Yuhta